src/sections/Community/Handbook/repo-data.js

```diff
@@ -301,6 +304,7 @@ export const repo_data = [
       },
       {
         project: "Service Mesh Patterns Action",
+        image: patterns,
```

Comment:
![image](https://user-images.githubusercontent.com/95926324/189519840-4746aaef-5f31-4555-a2f5-d8bfa575f458.png)

@leecalcote shouldn't we use  github actions here ?

Reply:
To be consistent in presentation with other GitHub Action projects that we have - yes, you’re right.

Reply:
Although, I really don't mind having the Service Mesh Patterns logo used a bit more than it is. 😀